modules/appeals_api/config/routes.rb

```diff
@@ -6,11 +6,13 @@
   namespace :v0, defaults: { format: 'json' } do
     resources :appeals, only: [:index]
     get 'healthcheck', to: 'appeals#healthcheck'
+    resources :claims, only: %i[index show]
   end
 
   namespace :docs do
     namespace :v0 do
-      resources :api, only: [:index]
```

Comment:
I think we may have consumers of the appeals API in production or close to it. We may need to bump the API version if we want to rename this URL, or preserve the `/api` route alongside the appeals one and label it deprecated.